src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -102,15 +112,6 @@ public User getUser(UUID id) {
     return null;
   }
 
-  /**
```

Comment:
I would actually leave this method here too in case there is a reason in the future that someone might already have a `User` object and want to add it.  It doesn't hurt to have both.